osu.Game/Tests/Visual/ScreenTestScene.cs

```diff
@@ -38,12 +38,11 @@ protected ScreenTestScene()
 
         private void addExitAllScreensStep()
         {
-            AddUntilStep("exit all screens", () =>
+            AddStep("exit all screens", () =>
             {
-                if (Stack.CurrentScreen == null) return true;
+                if (Stack.CurrentScreen == null) return;
 
                 Stack.Exit();
-                return false;
```

Comment:
Not a 100% about this change; it makes the tests run faster in browser but if anything ever breaks in screen exiting logic then we might not know about it (current version will only exit topmost if I understand correctly). Maybe this should unwind to the top in a manual `while` loop so that we know if it does - or maybe we just adjust step name so that this no longer claims to exit all screens?

Reply:
didn't mean to commit this here, will undo